week10/MovieViewer/app/src/main/java/com/wmw/movieviewer/repository/UserRepositoryImpl.kt

```diff
@@ -0,0 +1,10 @@
+package com.wmw.movieviewer.repository
+
+import com.wmw.movieviewer.helper.SharedPrefsManager
+
+class UserRepositoryImpl(private val sharedPrefsManager: SharedPrefsManager) : UserRepository {
```

Comment:
Not sure why you went with the right approach here, but decided to do things completely different in other objects/dependencies